src/mlpack/tests/hmm_test.cpp

```diff
@@ -1050,7 +1050,7 @@ TEST_CASE("GaussianHMMPredictTest", "[HMMTest]")
   {
     double loglikelihood;
     arma::vec forwardLogProb;
-    for (size_t t = 0; t<obs.n_cols; ++t)
+    for (size_t t = 0; t < obs.n_cols; ++t)
```

Comment:
That is a little overly stringent for my tastes.  That's more styling (of which I am not that as much a fan as other people) and more linting.

Reply:
Hehe, perhaps, it's been a part of the mlpack style guide for a long time: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#spacing-for-operators

So, maybe tedious, but better to have consistent code throughout.